website/src/contribute/release-guide.md

```diff
@@ -167,8 +167,23 @@ please submit your GPG public key into [MIT PGP Public Key Server](http://pgp.mi
 
 #### Website development setup
 
-Prepare to update the Beam website by following the [website development
-instructions]({{ site.baseurl }}/contribute/website-contributions/).
+Updating the Beam website requires access to both the man `apache/beam` repo and
+the `apache/beam-site` repo. The first contains most of the website sources,
+while the second contains reference manual generated from SDK code.
+
+You should already have setup access to `apache/beam`. Setting up access to
```

Comment:
Here and above, can you qualify what you mean by "access" to the repo? There are different permission sets granted to different groups:

* Anyone can submit a PR to the repo
* Committers can merge PRs and push to branches
* PMC can update various repo settings on GitHub